dist/exceptionless.d.ts

```diff
@@ -31,6 +31,10 @@ export interface IEnvironmentInfoCollector {
 export interface IErrorParser {
     parse(context: EventPluginContext, exception: Error): IError;
 }
+export interface IExitController {
```

Comment:
I don't think we need a global flag for this, do we? If we had an internal method that took a sync flag then we could just have the process and processSync methods both call that method. Would that work?